.circleci/src/workflows/harmony.yml

```diff
@@ -0,0 +1,16 @@
+when: << pipeline.parameters.run-harmony-workflow >>
```

Comment:
Having this file without `jobs` is breaking ci. We should probably delete this file for now